lib/chef/knife/ec2_server_create.rb

```diff
@@ -212,11 +212,18 @@ class Ec2ServerCreate < Knife
         :description => 'S3 URL (e.g. s3://bucket/file) for the encrypted_data_bag_secret_file',
         :proc => lambda { |url| Chef::Config[:knife][:s3_secret] = url }
 
-      option :json_attributes,
+      option :first_boot_attributes,
```

Comment:
@adamedx , the option name is `first_boot_attributes` in `chef`. That's why we have renamed `json_attributes` to `first_boot_attributes`. Is it ok to merge this?